src/tests/playwright/pages/perf-gain.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <script defer type="module">
+      import record from './processors/recorder/recorder-main.js';
+      import { beCloseTo, test, evaluate } from './util/audit.js';
+
+      test((async () => {
+        const ctx = new AudioContext({sampleRate: 48000});
+        const src = new ConstantSourceNode(ctx);
+        const gs = Array(100).fill().map(() => new GainNode(ctx));
+        const length = 1; // second
+        const {recorder, buffer} = await record(ctx, length);
+        gs.forEach((g, i) => i !== 0 && gs[i - 1].connect(g));
+        src.connect(gs[0]);
+        gs.at(-1).connect(recorder).connect(ctx.destination);
```

Comment:
This is interesting; what does `-1` index do here?

Reply:
Your comment above was really helpful for me to understand what's going on. Please add it to the code.